### DISCUSSION TITLE
从edge换到chrome后直播间全屏突然变成这样。弹出这个错误

### DISCUSSION THREAD

**Discussion by Godwin9527** [the1812/Bilibili-Evolved#4441] [提问 / Q&A] [answered]
![QQ截图20230917132525](https://github.com/the1812/Bilibili-Evolved/assets/86955226/76e6e572-3083-4534-9262-099b9707479a)
![QQ截图20230917132533](https://github.com/the1812/Bilibili-Evolved/assets/86955226/46b872d4-ed75-4b1c-b423-888786d5b54b)

**Comment by Geeyun-JY3** [comment, marked as answer]
先检查脚本本体和相关组件是否均已是最新版，若是并仍有相同现象，建议去开 issue，按要求填写表单提供更多信息。

**Reply by Godwin9527** [reply]
我更新了预览版就好了，谢谢
